src/ceph.in

```diff
@@ -849,7 +859,12 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, filter=True)
+
+    # implement "tell module help"
+    if childargs and len(childargs) >= 3 and childargs[0] == "tell" and childargs[2] == "help":
```

Comment:
it will breaks when childargs is None. It's better to do this check.

```
In [4]: a = None

In [5]: len(a)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-af8c77e09569> in <module>()
----> 1 len(a)

TypeError: object of type 'NoneType' has no len()
```